tools/update.hpp

```diff
@@ -108,53 +110,51 @@ namespace antler {
       }
 
       update_project(CLI::App& app) {
-         path = system::fs::current_path().string();
          subcommand = app.add_subcommand("update", "Update an app, dependency, library or test in your project.");
 
-         subcommand->add_option("-p, path", path, "Path containing the project's yaml file.");
+         subcommand->add_option("-p,--path", path, "Path containing the project's yaml file.")->default_val(".");

```

Comment:
Same as add_to.hpp. There are two (too many?) flags for `antler-project update -p <path1> app -p <path2>`

Reply:
This is sort of an issue now:
```
beauty$ aproj add app --help
Add a new app to your project.
Usage: antler-proj add app [OPTIONS]

Options:
  -h,--help                   Print this help message and exit
  -n,--name TEXT REQUIRED     The name of the app to add.
  -l,--lang TEXT REQUIRED     Language this app will use.
  --comp TEXT                 Options for the compiler for this app.
  --link TEXT                 Options for the linker for this app.

Examples:
	antler-proj add app -p ./path-to-project/ -n MyApp -l C++ --comp -O2
	antler-proj add app -n MyApp -l C++ "\-O2 -WError" \\-s
```
Path (`-p`) shows up in the example, but not in the options.

I think we can "fix" this by using leap-CLI and `app_add->fallthorugh(true)`. But then we take on leap-CLI11 and whatever issues it has. I'm not sure what the best approach is here.